src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -142,11 +154,17 @@ public static class TextCatalog
             WordEmbeddingEstimator.PretrainedModelKind modelKind = WordEmbeddingEstimator.PretrainedModelKind.SentimentSpecificWordEmbedding)
             => new WordEmbeddingEstimator(Contracts.CheckRef(catalog, nameof(catalog)).GetEnvironment(), outputColumnName, inputColumnName, modelKind);
 
-        /// <include file='doc.xml' path='doc/members/member[@name="WordEmbeddings"]/*' />
+        /// <summary>
+        /// Create an <see cref="WordEmbeddingEstimator"/>, which is a text featurizer that converts vectors
+        /// of text into numerical vectors using pre-trained embeddings models.
+        /// </summary>
         /// <param name="catalog">The text-related transform's catalog.</param>
-        /// <param name="customModelFile">The path of the pre-trained embeedings model to use. </param>
-        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
-        /// <param name="inputColumnName">Name of the column to transform.</param>
+        /// <param name="customModelFile">The path of the pre-trained embeddings model to use.</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.
+        /// This column's data type will be a vector of <see cref="System.Single"/>.</param>
+        /// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>,
+        /// the value of the <paramref name="outputColumnName"/> will be used as source.
```

Comment:
>source [](start = 81, length = 6)

let's not use 'source'. just drop 'as source'. it reads fine without it.

Reply:
I can modify it here, but it's everywhere, that's the pattern that a lot of our transforms follow.

---
In reply to: [277032960](https://github.com/dotnet/machinelearning/pull/3418#discussion_r277032960) [](ancestors = 277032960)